src/OceanSeaIceModels/PrescribedAtmospheres.jl

```diff
@@ -102,7 +102,7 @@ const CP{FT} = ConstitutiveParameters{FT} where FT
 @inline gas_constant(p::CP)   = p.gas_constant
```

Comment:
This is called `universal_gas_constant` now.

Reply:
I can't find that; I still see `Thermodynamics.Parameters.gas_constant`

Reply:
okie